moped-editor/src/views/projects/projectView/ProjectActivityLog.js

```diff
@@ -321,7 +321,8 @@ const ProjectActivityLog = () => {
                           "moped_proj_notes",
                           "moped_proj_components",
                           "moped_project_types",
-                          "moped_proj_personnel"
+                          "moped_project_files",
+                          "moped_proj_personnel",
                         ].includes(change.record_type) ? (
```

Comment:
i believe we can now do away with this special case handling—all the event triggers have been migrated to this new pattern 🎉 

this is fine as-is: we will clean this up in https://github.com/cityofaustin/atd-data-tech/issues/10897